### PR TITLE
#625 PR errors are corrected

### PR DESCRIPTION
**Pull request by Olcaey**
bot_like and search_blacklist_hashtags_in_media errors are corrected.
Please let me know if there is any other error needs to be fixed.

### CODE REVIEW

File: instabot/bot/bot_filter.py [hunk 1]
```diff
@@ -95,10 +95,17 @@ def search_stop_words_in_user(self, user_info):
 
 
 def search_blacklist_hashtags_in_media(self, media_id):
-    text = self.get_media_info(media_id)[0]['caption']['text']
+    media_info = self.get_media_info(media_id)
+    if media_info[0]['caption'] is None:
```
Comment (ademaro) [inline instabot/bot/bot_filter.py]: Why do not you use a single-line expression? `text = media['caption']['text'] if media['caption'] else ''` ?

Comment (Olcaey) [reply instabot/bot/bot_filter.py]: thanks, done.

**Comment by ohld** [comment]
Looks nice to me. Merge?

**Comment by ohld** [comment]
Can you please rebase the PR?

**Review by ohld — APPROVED** [review]

**Pull request merged**